src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -481,7 +481,7 @@ class ChatSwitcherView extends React.Component {
                     onConfirmUsers={this.startGroupChat}
                 />
 
-                {this.state.usersToStartGroupReportWith.length === MAX_GROUP_DM_LENGTH
+                {this.state.usersToStartGroupReportWith.filter(v => v.login !== this.props.session.email).length === MAX_GROUP_DM_LENGTH
```

Comment:
Left a comment on the original issue. But I believe that we will want to fix the problem even before we get to this point since the filter here is only necessary because a user is able to add themselves to a group chat to begin with. Their presence is implied in the group chat so we should not be able to select them at all.

Reply:
I gotcha. I got the issue mixed up. I had thought we didnt want to count self towards the count but the real issue was to not allow self to be added. I saw the solution provided by dsmith111 and that is the way to go.